pkgs/tools/X11/wpgtk/default.nix

```
@@ -50,5 +50,6 @@ python3Packages.buildPythonApplication rec {
     license = licenses.gpl2Only;
     platforms = platforms.linux;
     maintainers = [ maintainers.melkor333 maintainers.cafkafk ];
```

Perhaps change this while you're at it? :)
```suggestion
    maintainers = with maintainers; [ melkor333 cafkafk ];
```